conftest.py

```diff
@@ -530,6 +530,14 @@ def file_connections_data(request):
     return request.param
 
 
+@pytest.fixture()
+def load_keytab(tmp_path_factory):
```

Comment:
```suggestion
def create_keytab(tmp_path_factory):
```
Вроде тут ничего никуда не загружается